Readme.ja.md

```diff
@@ -0,0 +1,45 @@
+# Open棟梁
+”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
+
```

Comment:
@daisukenishino 

Please add link to English version.
See [readme file in OpenTouryoTempates repository](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2015/Readme.ja.md).
```md
このファイルの英語版は[こちら](Readme.md)から。
```